src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -264,6 +264,7 @@
     public static final String SHOW_FILE_LINKS_UPGRADE_WARNING = "showFileLinksUpgradeWarning";
     public static final String SIDE_PANE_WIDTH = "sidePaneWidthFX";
     public static final String LAST_USED_EXPORT = "lastUsedExport";
+    public static final String LAST_USED_CLIPBOARD_EXPORT = "lastUsedClipboardExport";
```

Comment:
I'm not sure but would expect that we can simply use the already existing `LAST_USED_EXPORT` (that the data is exported to the clipboard is not so important in my opinion). @JabRef/developers thoughts?

Reply:
I think it could go either way, though I do think it feels weird to be able to change/ use a preference in two different places for slightly different purposes.  However, one would assume that the user is exporting to the same format for both use cases more often than not.  I am not a regular user of JabRef, though, and I am interested to see what @fspreng thinks.

Reply:
Many thanks for implementing the suggested improvement so quickly!
For my colleagues and me, both ways would work. But as @tobiasdiez already mentioned, it is the very same export functionality that is called, just from a different menu. Therefore, it seems more reasonable to me to use the already existing preference instead of introducing an additional one. By the way, the fact that the same export functionality is used in both cases would be even more obvious to the user - or at least to me - if the lists of available export formats displayed as parts of the dialog windows entitled "Save as" ("Export"/"Export selected entries") and "Select export format" ("Export to clipboard") were sorted in an identical way. Right now, the available formats are listed in alphabetical order in the case of "Export to clipboard", while it is not in the case of "Export"/"Export selected entries".